openedx/features/edly/utils.py

```diff
@@ -363,7 +363,7 @@ def get_current_site_invalid_certificate_context(default_html_certificate_config
         return context
 
     context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
-    context['logo_src'] = get_logo_url()
+    context['logo_src'] = current_site_configuration.get_value('logo_image_url', '')
```

Comment:
@taimoor-ahmed-1 @mamjad00 Are we always going to provide absolute URL's for `logo_image_url` in site configuration?
Also is this information provided in the cloud client onboarding document?

Reply:
@zubair-arbi I looked up in the onboarding document and it doesn't contain `logo_image_url` in site configurations. 
Alternatively, should we use `logo` from `BRANDING` which is already available as absolute URL?
Also, there is a related bug with `valid` certificates which should be addressed: https://edlyio.atlassian.net/browse/EDLY-2440?focusedCommentId=25777